app/uk/gov/hmrc/fhregistrationfrontend/views/enrolment_pending.scala.html

```diff
@@ -15,47 +15,57 @@
  *@
 
 @import uk.gov.hmrc.fhregistrationfrontend.config.AppConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.fhregistrationfrontend.views.html.layout
+@import uk.gov.hmrc.fhregistrationfrontend.views.ViewHelpers
 
-@this(
-    mainTemplate: uk.gov.hmrc.fhregistrationfrontend.views.html.main_template_old
-)
+@this(layout: layout, viewHelpers: ViewHelpers)
 
 @()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
-@stylesheets = {<link rel="stylesheet" href='@routes.Assets.at("stylesheets/status.css")'/>}
 
-@title = @{
-  Messages(s"fh.status.received.title")
+@title1 = @{
+  Messages("fh.status.received.title")
 }
 
-@mainTemplate(title = title, linkElement = Some(stylesheets)) {
+@layout(title = title1) {
 
-  <div class="govuk-box-highlight appstatus">
-    <h1 class="heading-xlarge">
-      @title
-    </h1>
-  </div>
+    @viewHelpers.govukPanel(Panel(
+        title = Text(Messages("fh.status.received.title")),
+        content = HtmlContent("")
```

Comment:
I think if there is no content , you can remove content entirely

Reply:
Updated - I was having difficulties earlier, but I believe it was a syntax error.